core/finality-grandpa/src/communication/mod.rs

```diff
@@ -55,8 +54,7 @@ mod periodic;
 #[cfg(test)]
 mod tests;
 
-/// The consensus engine ID of GRANDPA.
-pub const GRANDPA_ENGINE_ID: ConsensusEngineId = [b'a', b'f', b'g', b'1'];
```

Comment:
@rphmeier see. Resolved.

Reply:
Doesn't answer the rename to `FRNK`.

Reply:
meh, just a string identifier. what does it matter?

Reply:
(it wasn't done as a rename; i hadn't realised that the original existed before i placed this one. but the original was in the wrong place anyway - it needed to be lower level to be useful.)